schemas/questionnaire_v1.json

```diff
@@ -141,48 +113,71 @@
             "$ref": "common_definitions.json#/id"
           },
           "title": {
-            "type": "string"
-          },
-          "hide_in_navigation": {
-            "type": "boolean"
+            "type": "string",
+            "description": "Title used for the navigation container."
           },
-          "skip_conditions": {
-            "$ref": "common_definitions.json#/skip_conditions"
-          },
-          "routing_rules": {
-            "ref": "common_definitions.json#/routing_rules"
+          "title_from_answers": {
+            "type": "array",
```

Comment:
When would this be used?

Reply:
In repeating question groups. I.e. where a section is created for each household member the title would be made up of each member's first and last name.

Reply:
Ahh yeah 🙄